menu.h

```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+#include "utils.h"
+
+using namespace std;
+namespace menu
+{
+    void menu()
+    {
+        cout << "Welcome to the game!" << endl;
+        cout << "Please select an option:" << endl;
+        cout << "1. Start Game" << endl;
+        cout << "2. Settings" << endl;
+        cout << "3. Quit" << endl;
+        int textsize=16;
```

Comment:
I believe we need to move `text_size` and `font_weight` outside of the function for this to work properly.

Reply:
done